stack.yaml

```diff
@@ -2,7 +2,7 @@
 # For more information, see: http://docs.haskellstack.org/en/stable/yaml_configuration/
 
 # Specifies the GHC version and set of packages available (e.g., lts-3.5, nightly-2015-09-21, ghc-7.10.2)
-resolver: lts-11.5
+resolver: 16.31
```

Comment:
I'm surprised by the naming change. Is it the new habit?

Reply:
Nope, it's just me who's tired. It wouldn't build like that, but we don't have a CI setup for this project, thank god for your keen eyes.